src/Screens/MyPage/CategoryScreen.tsx

```
@@ -0,0 +1,175 @@
+  const EmptySection = useCallback(
+    ({ title }: { title: string }) => (
+      <View style={styles.emptyContainer}>
+        <Text color={Color.gray3}>{title}</Text>
+      </View>
+    ),
+    [],
+  );
+
+  // 카테고리 item
+  const renderItem = useCallback(({ item, drag, isActive }: RenderItemParams<ListItem>) => {
+    const animatedStyle = useAnimatedStyle(() => {
```

ESLint

[error] React Hook "useAnimatedStyle" cannot be called inside a callback. React Hooks must be called in a React function component or a custom React Hook function.

(react-hooks/rules-of-hooks)